src/gen_rpc_acceptor.erl

```diff
@@ -197,7 +197,13 @@ call_worker(Parent, WorkerPid, Ref, M, F, A) ->
     % and manifest as timeout. Wrap inside anonymous function with catch
     % will crash the worker quickly not manifest as a timeout.
     % See call_MFA_undef test.
-    % Fun = fun() -> catch erlang:apply(M, F, A) end,
-    PacketBin = erlang:term_to_binary({WorkerPid, Ref, catch erlang:apply(M, F, A)}),
+    % rpc hides throw. We marked throw same as exit.
+    Ret = try erlang:apply(M, F, A) 
+          catch 
+               throw:Term -> {badrpc, {'EXIT', Term}};
```

Comment:
Agree. Throw throws depending on context, should let that to the user to decide.